src/module_PI/Arduino_Uno/SerialComm.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+  SerialControl.h - A class designed around serial control of the Alarm System
+*/
+#include "Arduino.h"
+#include "SerialComm.h"
+#include "DataPacket.h"
+
+#define SIZE_OF_HASH_IN_PACKET 8
+#define SIZE_OF_DATAPACKET_IN_BYTES 64
+
+void SerialComm::sendLogMessage(String logMessage){
+  DataPacket logPacket(logMessage, "LOG");
```

Comment:
Since these same Strings (LOG, COMMAND, etc.) are used in multiple classes it might be a good idea to have them pulled into an enum that can be accessed by both the SerialComm class and the DataPacket class.